docs/xks/developer-guide/scheduling-scaling.md

```diff
@@ -55,7 +55,67 @@ CPU limits should be treated slightly differently from memory limits. When memor
 
 <!-- TODO: Something about detecting CPU throtlling and adjusting CPU requests -->
 
-<!-- TODO: Something about VPA or link to how to use VPA -->
+## Vertical Pod Autoscaler
+
+Defining a good resource definitions is one of the hardest things that you can do in Kubernetes.

```

Comment:
Use either-
- Defining good resource definitions is
OR
- Defining a good resource definition is